core/trino-spi/src/main/java/io/trino/spi/security/SystemAccessControl.java

```diff
@@ -258,6 +258,17 @@ default Set<String> filterCatalogs(SystemSecurityContext context, Set<String> ca
      *
      * @throws AccessDeniedException if not allowed
      */
+    default void checkCanCreateSchema(SystemSecurityContext context, CatalogSchemaName schema, Map<String, Object> properties)
+    {
+        checkCanCreateSchema(context, schema);
```

Comment:
nit: Can we directly invoke `denyCreateSchema` so when we remove the deprecated API we don't have to modify this code.